lib/ffmpeg/util.py

```diff
@@ -66,3 +66,47 @@ def map_best_hw_format(format, hwformats):
   return mapformat(
     match_best_format(
       format, set(hwformats) & set(get_supported_format_map().keys())))
+
+@memoize
+def ffmpeg_cmd():
+  from lib.platform import info
+  if info()["os"] == "windows":
+    return 'ffmpeg.exe'
+  else:
+    return 'ffmpeg'
+
+@memoize
+def ffprobe_cmd():
+  from lib.platform import info
+  if info()["os"] == "windows":
+    return 'ffprobe.exe'
+  else:
+    return 'ffprobe'
+
+@memoize
+def wsl_path_convert(file_path):
+  from lib.platform import info
+  if info()["os"] == "windows":
+    # for the WSL will automation mount windows file system to /mnt/DRIVER/file/path
+    # windows execute file need to access really windows file path is DRIVER:\file\path
+    wsl_file_path = file_path.replace('/mnt/', '')
+    wsl_file_path = wsl_file_path.replace('/', '\\\\')
+    if wsl_file_path[1] == '\\':
+      wsl_file_path = wsl_file_path[0] + ':' + wsl_file_path[1:]
+    return wsl_file_path
+  else:
+    return file_path
+
+@memoize
+def wsl_device(format_string):
+  from lib.platform import info
+  if info()["os"] == "windows":
+    return ''
+  return format_string
+
+@memoize
+def wsl_format_covert(device):
+  from lib.platform import info
+  if info()["os"] == "windows" and device.endswith('va'):
+    return device.replace('va', '')
+  return device
```

Comment:
unnecessary (see earlier comment).

Reply:
hmm... actually `d3d11va` does not work in my suggestion.  But we can solve this differently by adding additional subclass requirement variable.